euscollada/test/euscollada-model-conversion-test.l

```diff
@@ -73,9 +78,10 @@
 
   (eusmodel-validity-check-one *samplerobot*)
   (send *samplerobot* :init-pose)
+  (send *samplerobot* :move-coords (make-coords) (send (car (send *samplerobot* :links)) :worldcoords))
   (assert (eps= 1.308000e+05 (send *samplerobot* :weight))
           ";; sample robot has not valid weight ~A" (send *samplerobot* :weight))
-  (assert (eps-v= #f(2.32416 0.0 791.942) (send *samplerobot* :centroid))
+  (assert (eps-v= #f(2.32416 0.0 68.4419) (send *samplerobot* :centroid))
```

Comment:
why we changed this? because of change in OpenHPP3 ? if so please add comment to refer openhrp3 github commits.

Reply:
This relates to ```(send *samplerobot* :move-coords (make-coords) (send (car (send *samplerobot* :links)) :worldcoords))```
It aims to fix origin of robot model.
For example, origin of models from OpenHRP3 is in foot level and origin of models from urdf is in root link.

After this PR, I would like to replace euscollada for using ROS collada/urdf parser.
ROS parser have no information for pose of root link.
So, this is required to pass test with next version.